### PR TITLE
Use bluebird promise.map to avoid overwhelming the DB in getSyncErrors

### PR DESCRIPTION
**Pull request by anotherminh**
Closes https://github.com/transcend-io/wildebeest/issues/79

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -43,6 +43,7 @@
   },
   "dependencies": {
     "aws-sdk": "^2.687.0",
+    "bluebird": "=3.7.2",
```
Comment (anotherminh) [inline package.json:46]: I used the same version as backend

File: src/classes/Wildebeest.ts [hunk 2]
```diff
@@ -497,24 +498,28 @@ export default class Wildebeest<TModels extends ModelMap> {
...
-        getStringKeys(this.models)
-          .map((modelName) => this.models[modelName].tableName)
-          .concat(this.ignoredTableNames),
-      ),
-    ]);
-    return flatten(results);
+        );
+      },
+      {
+        concurrency: 100, // The concurrent limit to avoid overwhelming the DB
```
Comment (michaelfarrell76) [inline src/classes/Wildebeest.ts:511]: think this is low enough?

Comment (anotherminh) [reply src/classes/Wildebeest.ts:511]: I think it craps out locally at 200

Comment (michaelfarrell76) [reply src/classes/Wildebeest.ts:511]: sounds great then. lets do it and worst case we drop it later

**Pull request merged**